addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -597,9 +597,9 @@ export class OdooEditor extends EventTarget {
                     description: this.options._t('Insert a rating over 3 stars'),
                     fontawesome: 'fa-star-o',
                     callback: () => {
-                        let html = '\u200B<span contenteditable="false" class="o_stars o_three_stars">';
-                        html += Array(3).fill().map(() => '<i class="fa fa-star-o"></i>').join('');
-                        html += '</span>';
+                        let html = '\u200B<span contenteditable="false" class="o_three_stars">';
+                        html += Array(3).fill().map(() => '<i class="o_stars fa fa-star-o"></i>').join('');
+                        html += '</span>\u200B';
```

Comment:
Why do we need this change?
The fact of moving `o_stars` to the fontawesome themselves means the unique ids will be applied to them instead and that's going to break the feature: `web_editor/controllers/main.py`'s `update_stars` route expects to find them with the xpath `".//span[@id='checkId-%s']"` but now it won't find anything anymore. Besides, it expects to find a _group_ of stars, which signify a rating, rather than a single star which signifies nothing on its own.

Reply:
Ah I get it, you wanted the ids on the fontawesomes themselves to apply my solution which misunderstood where the issue was ^^' My bad, the best option is probably to simply to mark fontawesomes as explicitly never similar:
![image](https://github.com/odoo/odoo/assets/24205914/bfacea54-6d5c-481a-9a52-0eda699e6319)